tests/components/device_tracker/test_init.py

```diff
@@ -541,8 +542,113 @@ def test_see_state(self):
+        #                                     timedelta(seconds=0))
+        # assert len(config) == 1
+        #
+        # state = self.hass.states.get('device_tracker.dev1')
+        # attrs = state.attributes
+        # self.assertEqual(state.state, STATE_NOT_HOME)
+        # self.assertEqual(state.object_id, 'dev1')
+        # self.assertEqual(state.name, 'dev1')
+        # self.assertEqual(attrs['friendly_name'], 'dev1')
+        # self.assertEqual(attrs['battery'], 100)
+        # self.assertEqual(attrs['latitude'], 0.3)
+        # self.assertEqual(attrs['longitude'], 0.8)
+        # self.assertEqual(attrs['test'], 'test')
+        # self.assertEqual(attrs['gps_accuracy'], 1)
+        # self.assertEqual(attrs['source_type'], 'gps')
+        # self.assertEqual(attrs['number'], 1)
+        #
+        # self.hass.config.components = []
+
+        with assert_setup_component(1, zone.DOMAIN):
+            zone_info = {
+                'name': 'Home',
+                'latitude': 32.880837,
+                'longitude': 32.880837,
+                'radius': 250,
+                'passive': False
+            }
+
+            setup_component(self.hass, zone.DOMAIN, {
+                'zone': zone_info
+            })
+
+        scanner = get_component('device_tracker.test').SCANNER
+        scanner.reset()
+        scanner.come_home('dev1')
+
+        with patch('homeassistant.components.device_tracker.dt_util.utcnow',
+                   return_value=register_time):
+            with assert_setup_component(1, device_tracker.DOMAIN):
+                assert setup_component(self.hass, device_tracker.DOMAIN, {
+                    device_tracker.DOMAIN: {
+                        CONF_PLATFORM: 'test',
+                        device_tracker.CONF_CONSIDER_HOME: 59,
+                    }})
+
+        state = self.hass.states.get('device_tracker.dev1')
+        attrs = state.attributes
+        self.assertEqual(STATE_HOME, state.state)
+        self.assertEqual('dev1', state.object_id)
+        self.assertEqual('dev1', state.name)
+        self.assertEqual('dev1', attrs.get('friendly_name'))
+        self.assertEqual(32.880837, attrs.get('latitude'))
+        self.assertEqual(32.880837, attrs.get('longitude'))
+        self.assertEqual(0, attrs.get('gps_accuracy'))
+        self.assertEqual(device_tracker.SOURCE_TYPE_ROUTER,
+                         attrs.get('source_type'))
+
+        scanner.leave_home('dev1')
+
+        with patch('homeassistant.components.device_tracker.dt_util.utcnow',
+                   return_value=scan_time):
+            fire_time_changed(self.hass, scan_time)
+            self.hass.block_till_done()
+
+        state = self.hass.states.get('device_tracker.dev1')
+        attrs = state.attributes
+        self.assertEqual(STATE_NOT_HOME, state.state)
+        self.assertEqual('dev1', state.object_id)
+        self.assertEqual('dev1', state.name)
+        self.assertEqual('dev1', attrs.get('friendly_name'))
```

Comment:
Please leave the constants as the second parameter since this becomes a critical lint issue for normal `assert` with `pytest`

```
C: 30,11: Comparison should be log_err.call_count == 1 (misplaced-comparison-constant)
```

It will also make it much easier to spot the changes (only the coords is apparent now)